haystack/testing/preview/document_store.py

```diff
@@ -189,9 +189,9 @@ def test_eq_filter_table(self, docstore: Store, filterable_docs: List[Document])
         )
 
     @pytest.mark.unit
-    def test_eq_filter_tensor(self, docstore: Store, filterable_docs: List[Document]):
```

Comment:
The word `tensor` is nowhere used in the codebase, let's be consistent and use `embedding` like the field in the `Document` class

Reply:
I wonder if this indicates that the matrix was actually used on purpose. If you wanted to support all kinds of tensors, testing on a matrix instead of just a vector would make sense.
However, I still can't see a good reason for wanting tensor embeddings.

Reply:
Yep, at the moment `Document` supports audio and images but only as paths to files, but we should support explicitly their respective representations. In that case tensors would make sense I guess, and I would expect tests to be adjusted to use matrix when needed.